lib/.eslintrc.yaml

```diff
@@ -261,30 +254,3 @@ globals:
   module: false
   internalBinding: false
   primordials: false
-overrides:
-  - files:
-      - ./*/*.js
-      - ./*.js
-      - ./internal/child_process/*.js
-      - ./internal/cluster/*.js
-      - ./internal/debugger/*.js
-      - ./internal/events/*.js
-      - ./internal/fs/*.js
-      - ./internal/modules/*.js
-      - ./internal/per_context/*.js
-      - ./internal/perf/*.js
-      - ./internal/policy/*.js
-      - ./internal/process/*.js
-      - ./internal/readline/*.js
-      - ./internal/readme.md
-      - ./internal/repl/*.js
-      - ./internal/source_map/*.js
-      - ./internal/streams/*.js
-      - ./internal/test/*.js
-      - ./internal/test_runner/**/*.js
-      - ./internal/tls/*.js
-      - ./internal/util/parse_args/*.js
-      - ./internal/watch_mode/*.js
-      - ./internal/webstreams/*.js
-    rules:
-      comma-dangle: [error, always-multiline]
```

Comment:
Just wanted to know so by default even on single line objects/arrays we would have commas 
like [1,2,]?

Reply:
By default it inherits from https://github.com/nodejs/node/blob/757c10414776b6f035ced453599527a841f47d6e/.eslintrc.js#L137